### ISSUE TITLE
CPU frequency stuck at low value after uninstalling gnome extension

### ISSUE THREAD

**Issue by Coshibu** [konkor/cpufreq#143]
I have experimented with different gnome extensions to control the performance and battery life on my laptop including this one. 
I also installed:
https://extensions.gnome.org/extension/945/cpu-power-manager/ 

I partly run them at the same time, then uninstalled and reinstalled them. Now my CPUs are stuck at the minimum frequency and changing the governor has no more effect even with both extensions uninstalled.
```
`user@XPS:~$ cat /proc/cpuinfo | grep MHz

cpu MHz     : 800.039

cpu MHz     : 800.035
cpu MHz     : 800.037
cpu MHz     : 800.026
cpu MHz     : 800.008
cpu MHz     : 800.036
cpu MHz     : 800.040
cpu MHz     : 800.028
user@XPS:~$ for governor in $(ls /sys/devices/system/cpu/cpu*/cpufreq/scaling_governor); do cat $governor; done
performance
performance
performance
performance
performance
performance
performance
performance`
```

**Comment by konkor** [comment]
@Coshibu You should not use multiple power management tools at the same time. It could create races for the access to CPU. Otherwise, you should be sure what each tool does and responsible for.

The CPUFREQ extension doesn't have any active services after the uninstalling it at least yet. But it leave some passive components and helper like cpufreqctl in /usr/bin or the policy or cpufreq font.

BTW Looks like you are using `intel_pstate` driver. So you should check /sys/devices/system/cpu/intel_pstate folder for MIN/MAX performance counters and monitor it for the process that is changing it.

Good luck! All be fine.

**Comment by Coshibu** [comment]
Yes, one shouldn't use multiple power management tools at the same time... Learned that one now. Not blaming you guys.  

It seems like /sys/devices/system/cpu/intel_pstate MIN/MAX changes don't take effect anymore. I can set the MIN to 80 and the CPU still runs at 800 Mhz. 

Is there a proper way to reset/reinstall the intel_pstate ?

**Comment by konkor** [comment]
@Coshibu Unfortunately, there is no some universal method to reset/uninstall/delete relaited kernel's power management services and their configs. The /sys/devices/system/cpu is just an API for kernel modules (drivers). So it could be a better solution for you to reinstall the system.

Actually, it's unusual situation. You can check /sys/devices/system/cpu/cpu0/cpufreq/scaling_max_freq and/or
/sys/devices/system/cpu/cpu0/cpufreq/cpuinfo_max_freq. Set it to the max available frequency from /sys/devices/system/cpu/cpu0/cpufreq/scaling_available_frequencies. If it will be changed in some time you should monitor for the process which changed it (you could use `lsof` command for that).

```sh
watch -n 1 lsof /sys/devices/system/cpu/cpu0/cpufreq/scaling_max_freq

## OR/AND FOR INTEL_PSTATE's SO
watch -n 1 lsof /sys/devices/system/cpu/intel_pstate/max_perf_pct
```

**Comment by konkor** [comment]
There are a lot of power management tools now days. It could be TLP, laptop-tools, system76-power and more. They don't have even GUI for management and configuration.

**Comment by Coshibu** [comment]
After checking different options in grub with no result, I booted a clean ubuntu image from USB. The CPU frequency was still at 800 Mhz indication that it wasn't OS related. A quick search yielded that this is a Dell safeguard mechanism to protect the laptop and is triggered by a thermal event, bad battery, or faulty charger. In my case, it was probably a thermal event I had at the same time I experimented with the above-mentioned extensions. My laptop failed to sleep and went full throttle in my rucksack sometime last week.

The solution was to unscrew the back-lid and disconnect the battery for >15 seconds. This resets the safeguard mechanism.
